sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -352,6 +358,15 @@ tableProvider
     : USING multipartIdentifier
     ;
 
+createTableClauses
```

Comment:
how about
```
createTableClause
  : (OPTIONS options=tablePropertyList) | ...
  ;
```
Then we can reuse it in more places
```
createTableHeader ... createTableClause*                              #createTable
createTableHeader ... (createTableClause | rowFormat | createFileFormat | ...)* . #createHiveTable
replaceTableHeader ... createTableClause*
```

Reply:
Good idea!